### DISCUSSION TITLE
jaxlib version for cuda 11.7

### DISCUSSION THREAD

**Discussion by ksapozhn** [jax-ml/jax#11976] [Q&A] [answered]
Is there a jaxlib version supporting cuda 11.7?

**Comment by ksapozhn** [comment, marked as answer]
I managed. issues is solved. thanks!

**Reply by hawkinsp** [reply]
And for anyone who comes across this question later: the jaxlib wheels are compatible across CUDA versions. Just install the 11.4 wheel, for example.

**Reply by ksapozhn** [reply]
Hi, what do you mean? i used
pip3 install jax[cuda] -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
what do you suggest to install instead?

**Reply by hawkinsp** [reply]
I'm saying: "that works". That's (at the moment) a CUDA 11.4 wheel. It will work on CUDA 11.7.
